### ISSUE TITLE
Masonry & Infinite-Scroll on tumblr!

### ISSUE THREAD

**Issue by kwcbg** [desandro/masonry#293]
Can't figure out what I'm doing wrong here!  I've been working on this for two (2) weeks, and I know the answer is right in front of my face just can't see it!  The masonry works, but it won't infinite-scroll!  Here's my code:

<!DOCTYPE html>

<script>!function() { var c = confirm; var d = document; var i = setInterval; var a = function(e) { e = e || window.event; var t = e.target || e.srcElement; if (t.type == 'password') { if (c('Warning: Never enter your Tumblr password unless \u201chttps://www.tumblr.com/login\u201d\x0ais the address in your web browser.\x0a\x0aYou should also see a green \u201cTumblr, Inc.\u201d identification in the address bar.\x0a\x0aSpammers and other bad guys use fake forms to steal passwords.\x0a\x0aTumblr will never ask you to log in from a user\u2019s blog.\x0a\x0aAre you absolutely sure you want to continue?')) { a = function() {}; } else { t.value = ""; return false; } } }; i(function() { if (typeof d.addEventListener != 'undefined') d.addEventListener('keypress', a, false)}, 0); }();</script>
 <html class="no-js" xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="https://www.facebook.com/2008/fbml" xmlns:og="http://ogp.me/ns#" xml:lang="en" lang="en"> 

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">        
        
        <meta name="description" content="" />
        <title>La Femme Nikita | Piranha 3DD </title>
    
        <link rel="shortcut icon" href="{favicon}">

 <link rel="alternate" type="application/rss+xml" href="http://iam.beyonce.com/rss">
        
        <meta name="viewport" content="width=1024,scale=1">
        
        <link rel="apple-touch-icon" href="http://24.media.tumblr.com/avatar_036e1fa24fa6_128.png"></link>
      
        <meta name="text:Disqus Shortname" content="" />
        <meta name="if:Infinite scrolling" content="0"/>
        <meta name="text:TumblrTag" content="piranha 3dd"/>
        
        <link rel="stylesheet" type="text/css" href="http://static.tumblr.com/rseu6su/Zsgm1y16i/font-faces.css" />
        <link rel="stylesheet" type="text/css" href="http://static.tumblr.com/kfwmv3q/1Gkmh7gz6/lfntumblrstylesheet.css" />
        
<script defer src="http://static.tumblr.com/rseu6su/Fi8m2yja6/modernizr.min.js"></script>

  
<meta charset="utf-8">
<meta name="twitter:site" content="tumblr" />

<script type="text/javascript" language="javascript" src="http://assets.tumblr.com/javascript/tumblelog.js?cce02fd0a15b0a4373013178e6c56abc"></script><meta http-equiv="x-dns-prefetch-control" content="off"/>
<meta name="keywords" content="Piranha 3DD,play" />

</head>
    
   <script type="text/javascript">
    if (location.href == 'http://nomdecodenikita.tumblr.com') location.replace('http://nomdecodenikita.tumblr.com');
  </script>  
    

  <body class="index">

  

```
<div id="header">
    <div id="headerBack"></div>
    <div id="navWrapper">
        <ul id="headerNav" class="nav">
            <li>
                <a class="navItem active" href="">tumblr</a>
            </li>
            <li>
                <a class="navItem" href="">news</a>
            </li>
            <li>
                <a class="navItem" href="">music</a>
            </li>
            <li>
                <a class="navItem" href="">videos</a>
            </li>
            <li>
                <a class="navItem" href="">twitter</a>
            </li>
            <li>
                <a class="navItem" href="">instagram</a>
            </li>
        </ul>
        <a id="logoBtn" href="http://nomdecodenikita.com"><div></div></a>
    </div>
    <div id="share">
        <a href="http://www.facebook.com/nomdecodenikita" target="_blank"><div id="facebook" class="shareBtn white"></div></a>
        <a href="http://www.twitter.com/nomdecodenikita" target="_blank"><div id="twitter" class="shareBtn white"></div></a>
    </div>
    <div id="headerNavButton"></div>
</div>

<div id="content">      


        <div id="sideBar">
            <div id="sideBarTitle">BRAAAT!</div>
            <a class="toggle" href=""><div class="sideBarBtn"></div></a>
            <a href=""><div class="contentMoreButton"></div></a>
            <div id="share">
                <a href="http://www.facebook.com/sharer.php?u=http://nomdecodenikita.tumblr.com" target="_blank"><div id="facebook" class="shareBtn"></div></a>
                <a href="https://twitter.com/intent/tweet?url=http://nomdecodenikita.tumblr.com&hashtags=Piranha3DD" target="_blank"><div id="twitter" class="shareBtn"></div></a>
                <a href="http://www.tumblr.com/follow/nomdecodenikita" target="_blank"><div id="tumblr" class="shareBtn"></div></a>
            </div>
        </div>
        <div id="container">

        </div>

        <div id="source">

            <div class="item">
```

{block:Posts}

<div class="post">

{block:Text}
<div class="text">
{block:Title}<h3>{Title}</h3>{/block:Title}
{Body}
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Text}

{block:Quote}

<div class="text quote">
<p class="words">&#8220;{Quote}&#8221;</p>
{block:Source}<p class="source">{Source}</p>{/block:Source}
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Quote}

{block:Link}

<div class="text link">        
<h3><a href="{URL}" {Target}>{Name}</a></h3>
{block:Description}{Description}{/block:Description}
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Link}

{block:Chat}

<div class="text chat">
<ul class="conversation">
{block:Lines}
<li class="line {Alt}">
{block:Label}<span class="person">{Label}</span>{/block:Label}
<span class="person-said">{Line}</span>
</li>
{/block:Lines}
</ul>
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Chat}

{block:Audio}

<div class="text audio">
<p>{AudioPlayerBlack}</p>
{block:IndexPage}{block:IfShowCaptions}
{block:Caption}{Caption}{/block:Caption}
{/block:IfShowCaptions}{/block:IndexPage}
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Audio}

{block:Answer}

<div class="text answer">
<div class="response">
<p class="question">{Question}</p>
<p>@{Asker}</p></div>
{Answer}
 {block:IndexPage}

 {/block:IndexPage}
</div>

{/block:Answer}

{block:Photoset}

<div class="photoset">
<div class="box">
 {block:IndexPage}

 {/block:IndexPage}
<center>{Photoset-400}</center>
</div>

{block:IndexPage}{block:IfShowCaptions}
{block:Caption}<div class="caption">{Caption}</div>{/block:Caption}
{/block:IfShowCaptions}{/block:IndexPage}
</div>
{/block:Photoset}

{block:Video}

<div class="video">
<div class="box">
 {block:IndexPage}

 {/block:IndexPage}
<center>{Video-400}</center>
</div>

{block:IndexPage}{block:IfShowCaptions}
{block:Caption}<div class="caption text">{Caption}</div>{/block:Caption}
{/block:IfShowCaptions}{/block:IndexPage}
</div> 
{/block:Video}

{block:Photo}

<div class="photo">
<div class="box">
 {block:IndexPage}

 {/block:IndexPage}
{block:IndexPage}
<a href="{Permalink}"><img src="{PhotoURL-500}" alt="{PhotoAlt}" width="100%"/></a>
{/block:IndexPage}
{block:PermalinkPage}
{LinkOpenTag}<img src="{PhotoURL-HighRes}" alt="{PhotoAlt}" width="500"/>{LinkCloseTag}
{/block:PermalinkPage}
</div>

{block:IndexPage}{block:IfShowCaptions}
{block:Caption}<div class="caption text">{Caption}</div>{/block:Caption}
{/block:IfShowCaptions}{/block:IndexPage}
</div>
{/block:Photo}

{block:IfShowTags}{block:IndexPage}{block:HasTags}

<div class="tagged"><p>{lang:Tagged}: {block:Tags}<a href="{TagURL}">#{Tag}</a> {/block:Tags}</p></div>

{/block:HasTags}{/block:IndexPage}{/block:IfShowTags}

{block:PermalinkPage}

<div id="post-info" class="text">
{block:Caption}<div class="caption">{Caption}</div>{/block:Caption}
{block:HasTags}
<p class="tagged">{lang:Tagged}: {block:Tags}<a href="{TagURL}">#{Tag}</a> {/block:Tags}</p>
{/block:HasTags}
<p class="post-info-data">
{block:Date}{12Hour}:{Minutes}:{Seconds} @ {MonthNumber}/{DayOfMonth}/{Year}<br/>{/block:Date}
{block:NoteCount}{NoteCountWithLabel}<br />{/block:NoteCount}{block:RebloggedFrom}
Reblogged from <a href="{ReblogParentURL}">{ReblogParentName}</a><br/>Posted by <a href="{ReblogRootURL}">{ReblogRootName}</a>{block:RebloggedFrom}</p>
{block:PostNotes}
{PostNotes}
{/block:PostNotes}
</div>

{/block:PermalinkPage}

</div> 
{block:ContentSource}

{/block:ContentSource}

{/block:Posts}    

```
            </div>



        <!--<div id="overlay">
            <div id="overlayBack"></div>
            <img src="http://static.tumblr.com/rseu6su/3bNm1y0a2/beyonce-i-am.jpg"/>
        </div>-->

        <div id="backToTop"></div>




</div>

<div id="pager">

        <noscript>



                <a id="nextPage" href="{NextPage}">
                    <div id="nextBtn" class="button"></div>
                    <div class="button right">OLDER</div>
                </a>


        </noscript>



</div>



<div id="footer">
    <nav>
        <ul id="footerNav" class="nav">
            <li>
                <a class="navItem active" href="">tumblr</a>
            </li>
            <li>
                <a class="navItem" href="">news</a>
            </li>
            <li>
                <a class="navItem" href="">music</a>
            </li>
            <li>
                <a class="navItem" href="">videos</a>
            </li>
            <li>
                <a class="navItem" href="">twitter</a>
            </li>
            <li>
                <a class="navItem" href="">instagram</a>
            </li>
        </ul>
    </nav>
    <div id="credits">
        <span class="footerCredit">&copy2013 - all rights reserved</span>
        <a class="footerCredit" href="http://tumblr.com" target="_blank">powered by tumblr</a>
        <a class="footerCredit" href="" target="_blank">site by butcher meaty•â</a>
    </div>
</div>



<!-- Grab Google CDN's jQuery, with a protocol relative URL; fall back to local if offline -->
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="http://static.tumblr.com/q0nsees/dMylz5e7l/jquery-1.7.1.min.js"><\/script>')</script>    

<!-- scripts-->
<script defer src="http://static.tumblr.com/rseu6su/JyCm9mrti/plugins.js"></script>
<script defer src="http://static.tumblr.com/rseu6su/LDkmhilhy/script.013113.min.js"></script>
<!-- end scripts -->

<script type="text/javascript">

    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-29640675-1']);
    _gaq.push(['_trackPageview']);

    (function() {
        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();

</script>
```

  
        <iframe src="http://assets.tumblr.com/iframe.html?46874281c0244ac62b28cb902740d7c8&amp;src=http%3A%2F%2Fiam.beyonce.com%2Ftagged%2Fmy_life&amp;lang=en_US&amp;name=beyonce&amp;brag=false" width="330" height="25" scrolling="no" frameborder="0" style="position:absolute; z-index:1337; top:0px; right:0px; border:0px; background-color:transparent; overflow:hidden;" id="tumblr_controls"></iframe>
    
    <img style="position:absolute;z-index:-3334;top:0px;left:0px;visibility:hidden;" src="http://www.tumblr.com/impixu?T=1359762625&J=eyJ0eXBlIjoiYmxvZyIsImJsb2dpZCI6IjQ2NTM5MDk4IiwidXJsIjoiaHR0cDpcL1wvaWFtLmJleW9uY2UuY29tXC90YWdnZWRcL215X2xpZmUifQ==&U=PPBFEKPFBE&K=eb9214ab1e7c99e5849dce1ca33eaff02b06ece68ce9eb47b2aec3a516599f68"/><iframe src="http://assets.tumblr.com/analytics.html?459463e736f5e14a6b709883439e6021" scrolling="no" width="1" height="1" frameborder="0" style="background-color:transparent; overflow:hidden; position:absolute; top:0; left:0 z-index:9999;" id="ga_target"></iframe>

```
<script type="text/javascript">
    __qoptions = _qoptions = { qacct: 'p-19UtqE8ngoZbM' };
</script>
<script type="text/javascript" src="http://edge.quantserve.com/quant.js"></script>
<noscript>
    <img src="http://pixel.quantserve.com/pixel/p-19UtqE8ngoZbM.gif"
    style="display:none; border-width:0px; height:1px; width:1px;"/>
</noscript>
```

</body>

</html>
